tests/modules/gatk4/markduplicates/test.yml

```diff
@@ -10,8 +10,19 @@
       md5sum: bda9a7bf5057f2288ed70be3eb8a753f
     - path: output/gatk4/test.metrics
 
-- name: gatk4 markduplicates test_gatk4_markduplicates_multiple_bams
-  command: nextflow run tests/modules/gatk4/markduplicates -entry test_gatk4_markduplicates_multiple_bams -c tests/config/nextflow.config
+- name: gatk4 markduplicates test_gatk4_markduplicates_no_metrics
+  command: nextflow run tests/modules/gatk4/markduplicates -entry test_gatk4_markduplicates_no_metrics -c tests/config/nextflow.config
+  tags:
+    - gatk4/markduplicates
+    - gatk4
+  files:
+    - path: output/gatk4/test.bai
+      md5sum: e9c125e82553209933883b4fe2b8d7c2
+    - path: output/gatk4/test.bam
+      md5sum: bda9a7bf5057f2288ed70be3eb8a753f
+
```

Comment:
Something is off with the md5sums:

```
_________________________________ test session _________________________________
'/home/runner/pytest_workflow_6p1unom2/gatk4_markduplicates_test_gatk4_markduplicates_no_metrics/output/gatk4/test.bai' does not exist while it should
_________________________________ test session _________________________________
Observed md5sum 'None' not equal to expected md5sum 'e9c125e82553209933883b4fe2b8d7c2' for file '/home/runner/pytest_workflow_6p1unom2/gatk4_markduplicates_test_gatk4_markduplicates_no_metrics/output/gatk4/test.bai'
_________________________________ test session _________________________________
'/home/runner/pytest_workflow_6p1unom2/gatk4_markduplicates_test_gatk4_markduplicates_no_metrics/output/gatk4/test.bam' does not exist while it should
_________________________________ test session _________________________________
Observed md5sum 'None' not equal to expected md5sum 'bda9a7bf5057f2288ed70be3eb8a753f' for file '/home/runner/pytest_workflow_6p1unom2/gatk4_markduplicates_test_gatk4_markduplicates_no_metrics/output/gatk4/test.bam'
_________________________________ test session _________________________________
'gatk4 markduplicates test_gatk4_markduplicates_no_metrics' exited with exit code '1' instead of '0'.
=========================== short test summary info ============================
FAILED tests/test_versions_yml.py::test_ensure_valid_version_yml[gatk4 markduplicates test_gatk4_markduplicates_no_metrics]
FAILED tests/modules/gatk4/markduplicates/test.yml::gatk4 markduplicates test_gatk4_markduplicates_no_metrics::output/gatk4/test.bai::should exist
FAILED tests/modules/gatk4/markduplicates/test.yml::gatk4 markduplicates test_gatk4_markduplicates_no_metrics::output/gatk4/test.bai::md5sum
FAILED tests/modules/gatk4/markduplicates/test.yml::gatk4 markduplicates test_gatk4_markduplicates_no_metrics::output/gatk4/test.bam::should exist
FAILED tests/modules/gatk4/markduplicates/test.yml::gatk4 markduplicates test_gatk4_markduplicates_no_metrics::output/gatk4/test.bam::md5sum
FAILED tests/modules/gatk4/markduplicates/test.yml::gatk4 markduplicates test_gatk4_markduplicates_no_metrics::exit code should be 0
============= 6 failed, 14 passed, 477 skipped in 61.61s (0:01:01) =============
```